src/general/smoothing_kernels.jl

```diff
@@ -390,24 +388,19 @@ struct WendlandC2Kernel{NDIMS} <: AbstractWendlandKernel{NDIMS} end
 @fastpow @inline function kernel(kernel::WendlandC2Kernel, r::Real, h)
     q = r / h
 
-    result = (1 - q / 2)^4 * (2q + 1)
+    result = (1 - q / 2)^4 * (2 * q + 1)
 
     # Zero out result if q >= 2
     result = ifelse(q < 2, normalization_factor(kernel, h) * result, zero(q))
 
     return result
 end
 
-@fastpow @muladd @inline function kernel_deriv(kernel::WendlandC2Kernel, r::Real, h)
+@fastpow @inline function kernel_deriv(kernel::WendlandC2Kernel, r::Real, h)
```

Comment:
```suggestion
@inline function kernel_deriv(kernel::WendlandC2Kernel, r::Real, h)
```

its only used from ^4